### PR TITLE
[RDY] Start off with DR implementation in Neo4j.

### PR DESCRIPTION
**Pull request by gfokkema**

### CODE REVIEW

**Comment by pvanagtmaal** [comment]
Looking good. Will be a hard one to rebase on, so we probably need to plan the merging.

File: dnainator-core/src/main/java/nl/tudelft/dnainator/graph/impl/Neo4jAnnotation.java [hunk 1]
```diff
@@ -39,6 +42,16 @@ public Neo4jAnnotation(GraphDatabaseService service, Node delegate) {
...
 				)
 			);
+			this.mutations = new ArrayList<>();
+			delegate.getRelationships(Direction.OUTGOING, RelTypes.MUTATION).forEach(e -> {
+				Node end = e.getEndNode();
+				String name	= (String) end.getProperty(DRMutationProperties.ID.name());
+				String type	= (String) end.getProperty(DRMutationProperties.TYPE.name());
+				String change	= (String) end.getProperty(DRMutationProperties.CHANGE.name());
+				String filter	= (String) end.getProperty(DRMutationProperties.FILTER.name());
+				int pos		= (int) end.getProperty(DRMutationProperties.POSITION.name());
```
Comment (Balletie) [inline dnainator-core/src/main/java/nl/tudelft/dnainator/graph/impl/Neo4jAnnotation.java]: Spaces instead of tabs, before the `=`? :innocent:

**Comment by Balletie** [comment]
No further comments, apart from the stylistic comment.

**Comment by pvanagtmaal** [comment]
Looking good. Some minor issues are fixed in #162.

**Pull request merged**